armi/bookkeeping/db/__init__.py

```diff
@@ -86,6 +86,7 @@ def loadOperator(
     loadNode,
     statePointName=None,
     allowMissing=False,
+    handleInvalids=True,
```

Comment:
suggest rename since handle is unclear. "query invalids" or something like that. handel sounds like ignore.

Reply:
this is already an arg name used in ARMI and I was just exposing it further back in the chain. Renaming would be a bigger effort.